pkg/platform/runtime/target/target.go

```diff
@@ -74,8 +76,15 @@ func (t Trigger) IndicatesUsage() bool {
 	return t.IsExecTrigger() && funk.Contains(usageTriggers, TriggerExec)
 }
 
+var (
+	triggerExecPrefix     = TriggerExec.String() + ": "
+	triggerExecutorPrefix = TriggerExecutor.String() + ": "
+)
+
 func (t Trigger) IsExecTrigger() bool {
-	return strings.HasPrefix(string(t), string(TriggerExec)+": ")
+	trigger := t.String()
+	return strings.HasPrefix(trigger, triggerExecutorPrefix) ||
+		strings.HasPrefix(trigger, triggerExecPrefix)
```

Comment:
This logic seems to only cover the `state exec` command.. so we shouldn't address executors here.